AwsCryptographyPrimitives/test/TestKDF_TestVectors.dfy

```diff
@@ -357,6 +1238,33 @@ module TestKDFK_TestVectors {
     }
   }
 
+  // It is complicated to check every byte of ever vector
+  // This takes all the okm values and digests them.
+  // This way it is easy for us to say
+  // that JS and Dafny are testing the same test vectors.
+  // I only use okm because there is a deterministic relationship
+  // between all inputs and the okm.
+  const vectorOkmDigest: seq<uint8> := [
```

Comment:
It's probably cleaner to embedded these digests into the test vectors as an additional member/field. 

Otherwise, if we change one p, we need to update this list as well.